src/main/blackbox/blackbox.c

```diff
@@ -812,11 +812,7 @@ static bool sendFieldDefinition(const char * const *headerNames, unsigned int he
             return false; //Someone probably called us again after we had already completed transmission
         }
 
-        charsWritten = blackboxPrint("H Field ");
-        charsWritten += blackboxPrint(headerNames[xmitState.headerIndex]);
-        blackboxWrite(':');
-        charsWritten++;
-
+        charsWritten = blackboxPrintf("H Field %s:", headerNames[xmitState.headerIndex]);
```

Comment:
But the gain is negligible here (this code runs only few times per arming) and new version is much easier to read.
Also it may save few bytes of flash.

If speed is that important, it would be better to improve `printf` to use buffering or use write where possible. Serial port can then implement writeBytes virtual function and the gain may be noticeable.